dvc/progress.py

```diff
@@ -58,12 +62,10 @@ def __init__(
         kwargs  : anything accepted by `tqdm.tqdm()`
         """
         kwargs = kwargs.copy()
-        kwargs.setdefault("unit_scale", True)
         if bytes:
-            bytes_defaults = dict(
-                unit="B", unit_scale=True, unit_divisor=1024, miniters=1
-            )
-            kwargs = merge(bytes_defaults, kwargs)
+            kwargs = merge(self.BYTES_DEFAULTS, kwargs)
+        else:
+            kwargs.setdefault("unit_scale", total > 999 if total else True)
```

Comment:
what's the reason of moving this one below an `else` clause?

Reply:
Minor efficiency. Wouldn't do anything if the `if` block above was executed.